packages/server/modules/shared/authz.ts

```diff
@@ -309,21 +309,28 @@ export const authPipelineCreator = (
 
 const validateStreamPolicyAccessFactory =
   (deps: {
+    getStream: StreamGetter
     policyInvoker: (params: {
       authData: AuthData
       policies: Authz.AuthPolicies
     }) => Promise<Authz.AuthPolicyResult>
   }): AuthPipelineFunction =>
   async (authData) => {
-    const { context, params, authResult } = authData
+    const { context, params } = authData
 
-    if (authHasFailed(authResult)) return { context, authResult }
```

Comment:
sometimes auth pipeline steps are expected to break on previous failures, sometimes not 😢